internal/adapters/operation/operation_storage_redis.go

```diff
@@ -263,6 +265,34 @@ func (r *redisOperationStorage) ListSchedulerFinishedOperations(ctx context.Cont
 	return operations, nil
 }
 
+func (r *redisOperationStorage) CleanOperationsHistory(ctx context.Context, schedulerName string) error {
+	operationsIDs, err := r.getFinishedOperationsFromHistory(ctx, schedulerName, time.Time{}, time.Now())
+	if err != nil {
+		return err
+	}
+
+	var operationIDsKeys []string
+	for _, operationID := range operationsIDs {
+		operationIDsKeys = append(operationIDsKeys, r.buildSchedulerOperationKey(schedulerName, operationID))
+	}
+	if len(operationIDsKeys) > 0 {
```

Comment:
Since you already know the len of "operationsIDs", why don't you test it before trying to populate an array and then testing it? Also, an optimization coming from this would be using make to create an array with the size of len(operationIDs)